doc/Manuals/ug-syspar_x-road_v6_system_parameters.md

```diff
@@ -531,3 +535,46 @@ TLS_DHE_RSA_WITH_AES_256_CBC_SHA256
 > (see [*https://docs.oracle.com/javase/8/docs/technotes/guides/security/SunProviders.html#SunJSSEProvider*](https://docs.oracle.com/javase/8/docs/technotes/guides/security/SunProviders.html#SunJSSEProvider) for possible values)
 >
 > Note. OpenJDK 8 on RHEL 7 supports ECDHE key agreement protocol starting from RHEL 7.3. In RHEL 7 versions prior to RHEL 7.3 only DHE cipher suites are supported.
+
+## 6 Adding command line arguments
+
+Sometimes you may need to provide command line arguments for Security Server, Central Server or Configuration Proxy, in order to override some parameters. For example if you wish to increase Java's maximum heap size, you can do it with the properties file `/etc/xroad/services/local.properties`. The file is also included in the backup archive file when taking a configuration backup of either Central Server or Security Server.
+
+### 6.1 The configuration file
+
+The file `/etc/xroad/services/local.properties` replaces the file `/etc/xroad/services/local.conf` starting from X-Road version 7. 
+
+The `local.properties` file's contents consists of key-value pairs such as `XROAD_SIGNER_PARAMS=-XX:MaxMetaspaceSize=100m`. Notice that you cannot use quotes in the values. If you have multiple values for one key, use space to separate the values e.g. `XROAD_SIGNER_PARAMS=-Xmx100m -XX:MaxMetaspaceSize=100m`. Also notice that with `local.properties` you can only add new parameters and override old ones – you cannot remove existing default parameters.
+
+Example of `/etc/xroad/services/local.properties` with modifications that override the default Java memory parameters for Security Server:
+
+```
+XROAD_PROXY_PARAMS=-Xms200m -Xmx1024m
+```
+
+Example of `/etc/xroad/services/local.properties` with modifications that override the default Java memory parameters for Central Server:
+
+```
+XROAD_JETTY_PARAMS=-Xms200m -Xmx1024m
+```
+
+All possible properties to adjust in this file:
+
+| Property | Details |
+|--------------------|---------|
+| XROAD_SIGNER_PARAMS | Parameters for the Signer |
```

Comment:
XROAD_PARAMS (params applied for all processes) is missing

Reply:
Added